spec/classes/plugin/dns_rfc2136_spec.rb

```diff
@@ -19,7 +19,7 @@ class { 'letsencrypt':
         osrelease = facts[:os]['release']['major']
         osfull = "#{osname}-#{osrelease}"
         case osfull
-        when 'Debian-10', 'Debian-11', 'AlmaLinux-8', 'RedHat-8', 'Ubuntu-22.04', 'Ubuntu-20.04', 'Ubuntu-18.04', 'Fedora-36'
+        when 'Debian-10', 'Debian-11', 'AlmaLinux-8', 'RedHat-8', 'Ubuntu-22.04', 'Ubuntu-20.04', 'Ubuntu-18.04', 'Fedora-36', 'Rocky-8'
```

Comment:
at which point should this be the `default` case?

Reply:
when I switch to EL9 in the next-next PR